### PR TITLE
Event migration, interval label and variable

### PR DESCRIPTION
**Pull request by lboeman**
Adds a test event forecast object, and forecast data. This test data is added to support https://github.com/SolarArbiter/solarforecastarbiter-core/pull/361.
The event forecast metadata is as follows: 
```
{'name': 'Weather Station Event Forecast',
 'issue_time_of_day': '05:00',
 'lead_time_to_start': 60.0,
 'interval_length': 5.0,
 'run_length': 60.0,
 'interval_label': 'event',
 'interval_value_type': 'instantaneous',
 'variable': 'event',
 'aggregate': None,
 'forecast_id': '24cbae4e-7ea6-11ea-86b1-0242ac150002',
 'provider': 'Organization 1',
 'extra_parameters': ''}
```
I generated a random series of boolean values between 2019-04-14T07:00Z and 2019-04-17T06:55Z to match the index of the available test data for the "DA GHI" forecast (uuid: '11c20780-76ae-4b11-bef1-7a75bdc784e3') at the test "Weather Station" site.

### CODE REVIEW

File: datastore/migrations/0038_event_forecasts.down.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        SELECT id from arbiter_data.cdf_forecasts_groups WHERE interval_label = 'event')
+);
+
+-- Remove any event metadata
+DELETE FROM arbiter_data.forecasts WHERE interval_label = 'event';
+DELETE FROM arbiter_data.cdf_forecasts_groups WHERE interval_label = 'event';
+DELETE FROM arbiter_data.observations WHERE interval_label = 'event';
+DELETE FROM arbiter_data.aggregates WHERE interval_label = 'event';
+
+ALTER TABLE arbiter_data.forecasts CHANGE COLUMN interval_label interval_label ENUM('beginning', 'ending', 'instant', 'event') NOT NULL;
```
Comment (alorenzo175) [inline datastore/migrations/0038_event_forecasts.down.sql:15]: remove event to go back to what it was?

File: datastore/migrations/0038_event_forecasts.up.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+SET @site_id = UUID_TO_BIN('123e4567-e89b-12d3-a456-426655440001', 1);
+
+INSERT INTO arbiter_data.forecasts (
+    id, organization_id, site_id, name, variable, issue_time_of_day,
+    lead_time_to_start, interval_label, interval_length, run_length,
+    interval_value_type, extra_parameters, created_at, modified_at
+) VALUES (
+    @event_fx_id, @orgid, @site_id, "Weather Station Event Forecast", "event",
+    "05:00", 60.0, "event", 5, 60.0, "instantaneous", "",
+    TIMESTAMP('2020-04-15 00:00'), TIMESTAMP('2020-04-15 00:00')
```
Comment (alorenzo175) [inline datastore/migrations/0038_event_forecasts.up.sql:18]: should this be consistent with the values i.e. 2019?

File: datastore/migrations/0038_event_forecasts.down.sql (hunk 1 above)
Comment (lboeman) [reply datastore/migrations/0038_event_forecasts.down.sql]: Thanks. Careful of the fresh copy pasta, may present burning hazard.

**Comment by lboeman** [comment]
Took care of those two comments, @alorenzo175 any other concerns before this is merged?

**Pull request merged**